src/hooks/useWindowSize.ts

```diff
@@ -31,13 +31,7 @@ function getDimensions(): WindowDimensions {
  * @returns Dimensions of the window
  */
 export function useWindowSize(): WindowDimensions {
-  const [windowSize, setWindowSize] = useState<WindowDimensions>(() => {
-    if (typeof window !== 'undefined') {
-      return getDimensions();
-    } else {
-      return nullDimensions;
-    }
-  });
+  const [windowSize, setWindowSize] = useState<WindowDimensions>(nullDimensions);
```

Comment:
This makes the window size null on first render on client side too, which is not right.

Reply:
I think the useEffect sets it on mount. It's working in my example 5.